registry/registry-sofa/src/test/java/com/alipay/sofa/common/profile/StringUtil.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.common.profile;
+
+/**
+ * @author khotyn
+ * @version StringUtil.java, v 0.1 2023年07月05日 14:40 khotyn
+ */
+public class StringUtil {
```

Comment:
Haven't seen where use of the `com. alipay. sofa. common. profile. StringUtil` class in test.

Reply:
sofa-registry-test depends on this class.

Reply:
Related SOFARegistry issue: https://github.com/sofastack/sofa-registry/pull/332.

After SOFARegistry new version released, we can delete this class.